doc/man1/openssl.pod

```diff
@@ -647,6 +647,11 @@ the PKCS#11 URI as defined in RFC 7512 should be possible to use directly:
 
 =head2 Provider Options
 
+If any of these options are given, then the default location for
+ciphers and digests (e.g., EVP_get_cipherbyname(3) and
+EVP_get_digestbyname(3))
+are not used.
```

Comment:
I think this text is confusing. The real user visible effect is that the implementation of ciphers and digests from an engine if there is any configured is not used. Please adjust the text to something like: `If any of these options are given engine based implementations of ciphers and digests are not used.`

Reply:
I'm not sure if "engines not used" is the main user-visible effect. Loading the FIPS provider, for example, will make a whole bunch of common algorithms not used.

Reply:
But I leave this to @levitte to decide what wording to use.